tests/dag_block_test.cpp

```diff
@@ -218,8 +218,7 @@ TEST_F(DagBlockTest, overlap) {
 }
 
 TEST_F(DagBlockMgrTest, proposal_period) {
-  auto node_cfgs = make_node_cfgs(1);
-  FullNode::Handle node(node_cfgs[0]);
+  auto node = create_nodes(1).front();
```

Comment:
How are these changes related to removing "register_s_ptr and related stuff" ?

Reply:
Aaah I see now you replace FullNode::Handle , which is some weird derived shared_ptr. My bad